manifests/02_config.crd.yaml

```diff
@@ -28,6 +30,375 @@ spec:
             submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
           type: string
         metadata:
+          description: ObjectMeta is metadata that all persisted resources must have,
```

Comment:
Just confirming that the `crd-schema-gen` tool generated all of this, and it makes sense to include it even those these are all cluster-defined APIs?

See: https://github.com/kubernetes-sigs/controller-tools/pull/266

Reply:
Yeah it is
```
[jesusr@transam cluster-svcat-apiserver-operator{bz1710937}]$ git status
On branch bz1710937
nothing to commit, working tree clean                                                                                                                                                                                                                                           
[jesusr@transam cluster-svcat-apiserver-operator{bz1710937}]$ grep update Makefile                         
# It will generate targets {update,verify}-bindata-$(1) logically grouping them in unsuffixed versions of these targets                                                                                                                                                         
# and also hooked into {update,verify}-generated for broader integration.                                  
update-codegen-crds: CRD_SCHEMA_GEN_GOPATH :=$(shell mktemp -d)
update-codegen-crds: crd-schema-gen
.PHONY: update-codegen-crds
update-codegen: update-codegen-crds
.PHONY: update-codegen
[jesusr@transam cluster-svcat-apiserver-operator{bz1710937}]$ make update-codegen^C
[jesusr@transam cluster-svcat-apiserver-operator{bz1710937}]$ git status
On branch bz1710937
nothing to commit, working tree clean
[jesusr@transam cluster-svcat-apiserver-operator{bz1710937}]$ make update-codegen
git clone -b v1.0.0 --single-branch --depth 1 https://github.com/openshift/crd-schema-gen.git /tmp/tmp.hKTPv7gUKD/src/github.com/openshift/crd-schema-gen
Cloning into '/tmp/tmp.hKTPv7gUKD/src/github.com/openshift/crd-schema-gen'...
remote: Enumerating objects: 5463, done.                            
remote: Counting objects: 100% (5463/5463), done.                   
remote: Compressing objects: 100% (4366/4366), done.                
remote: Total 5463 (delta 858), reused 4329 (delta 643), pack-reused 0
Receiving objects: 100% (5463/5463), 12.66 MiB | 1.13 MiB/s, done.
Resolving deltas: 100% (858/858), done.
Note: checking out '2dd386fe49ae959ecabd912a0d93010f838faff1'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout. 

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

GOPATH=/tmp/tmp.hKTPv7gUKD GOBIN=/tmp/tmp.hKTPv7gUKD/bin go install /tmp/tmp.hKTPv7gUKD/src/github.com/openshift/crd-schema-gen/cmd/crd-schema-gen
/tmp/tmp.hKTPv7gUKD/bin/crd-schema-gen --apis-dir vendor/github.com/openshift/api/operator/v1
Copying "vendor/github.com/openshift/api/operator/v1/"* to temporary "/home/devel/jesusr/dev/src/github.com/openshift/cluster-svcat-apiserver-operator/605455380/pkg/apis"...
Validation of ServiceCatalogAPIServer.operator.openshift.io in manifests/02_config.crd.yaml did not change.
[jesusr@transam cluster-svcat-apiserver-operator{bz1710937}]$ git status
On branch bz1710937
nothing to commit, working tree clean
[jesusr@transam cluster-svcat-apiserver-operator{bz1710937}]$ make verify-codegen
git clone -b v1.0.0 --single-branch --depth 1 https://github.com/openshift/crd-schema-gen.git /tmp/tmp.4nejJVHKj3/src/github.com/openshift/crd-schema-gen
Cloning into '/tmp/tmp.4nejJVHKj3/src/github.com/openshift/crd-schema-gen'...
remote: Enumerating objects: 5463, done.                            
remote: Counting objects: 100% (5463/5463), done.                   
remote: Compressing objects: 100% (4366/4366), done.                
remote: Total 5463 (delta 858), reused 4329 (delta 643), pack-reused 0                                                                  
Receiving objects: 100% (5463/5463), 12.66 MiB | 2.29 MiB/s, done.
Resolving deltas: 100% (858/858), done.
Note: checking out '2dd386fe49ae959ecabd912a0d93010f838faff1'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout. 

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

GOPATH=/tmp/tmp.4nejJVHKj3 GOBIN=/tmp/tmp.4nejJVHKj3/bin go install /tmp/tmp.4nejJVHKj3/src/github.com/openshift/crd-schema-gen/cmd/crd-schema-gen
/tmp/tmp.4nejJVHKj3/bin/crd-schema-gen --apis-dir vendor/github.com/openshift/api/operator/v1 --verify-only
Copying "vendor/github.com/openshift/api/operator/v1/"* to temporary "/home/devel/jesusr/dev/src/github.com/openshift/cluster-svcat-apiserver-operator/391328167/pkg/apis"...
Validation of ServiceCatalogAPIServer.operator.openshift.io in manifests/02_config.crd.yaml did not change.
```